src/ToggleSwitch.tsx

```diff
@@ -121,6 +123,7 @@ export const ToggleSwitch = memo(
                     id={inputId}
                     title={inputTitle}
                     checked={props_checked ?? checked}
+                    {...(name !== undefined && { name: name })}
```

Comment:
```suggestion
                    name={name}
```
Although I like the spreading thechnique I this particular case it has no effect on the rendered html